### ISSUE TITLE
Muse Sounds "Women" instrument stops at G4

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#22572]
### Issue type

Muse Sounds bug

### Bug description

The new "Women" instrument seems to be missing all samples above G4, making its range *less* than the "Altos" when it should be *more*.

### Steps to reproduce

1. New score, SATB Closed Score
2. Enter a C major scale into top staff starting from middle C (C4) - C D E F G A B C
3. Play

Result: playback does not include notes after the G

### Screenshots/Screen recordings

[Screen recording 2024-04-25 7.22.10 AM.webm](https://github.com/musescore/MuseScore/assets/1799936/4af08e61-175e-47a6-9515-36e5adf3054d)

### MuseScore Version

OS: Debian GNU/Linux 12 (bookworm), Arch.: x86_64, MuseScore Studio version (64-bit): 4.3.0-241101558, revision: github-musescore-musescore-49f06da

### Regression

No.

### Operating system

Debian 12

### Additional context

To be clear, this isn't about extreme ranges, this is about half the normal range simply being missing.  I'm guessing it was meant to switch to soprano samples at that point but somehow those got left out?

**Comment by bkunda** [comment]
I am unable to reproduce this. @sampleeditor is it likely that this was fixed, but the fix hasn't yet been pushed to production?

**Comment by MarcSabatella** [comment]
FWIW, my Muse Choir reports 0.3.14, and my Muse Sampler is 0.6.1.162 (these are current in Linux "Muse Sounds Manager").  This was originally noted by others on Discord as well when Muse Choir was updated a couple of weeks ago, but we were hoping it just needed the Muse Sampler 0.6 update to start working.  That has proven not to be the case however, at least on Linux.

**Comment by Jojo-Schmitz** [comment]
Where do you see the version of the sounds?

**Comment by MarcSabatella** [comment]
On Linux, logs are printed to the console when Muse Sounds Managers starts up.  I assume this info is also recorded somewhere within the downloads folder but don’t know where.

**Comment by sampleeditor** [comment]
@MarcSabatella I can't seem to duplicate this on my end, but what you're describing means that the sopranos are not sounding from g#4 and up. Does the same thing occur with the Men in the lower octave?

**Comment by MarcSabatella** [comment]
No, "Men" works just fine, range is G1 - C5.  Also, Sopranos are fine, range is F3 - F6.  Altos are fine, range is D3 - A5.  Only Women seems messed up, range is only D3 - G4.  I just tested on Windows and it's exactly the same, also using Muse Sampler 0.6.1.162 and Muse Choir 0.3.14 (as reported by Muse Hub Beta while updating choir and shown in C:\ProgramData\MuseHub\Logs\current.txt).

So, for those of you who cannot reproduce - is that on a different Muse Sampler, or a different Muse Choir, or a different OS (I have no way of testing macOS, but see the issue on both Windows and Linux)?  I'll also check on Discord to see what the others who reported the same are using.

**Comment by MarcSabatella** [comment]
"Full choir" (just realized that's a thing too) has the same issue of not playing above G4.

**Comment by Jojo-Schmitz** [comment]
OK, I also got Choir 0.3.13

**Comment by MarcSabatella** [comment]
I checked, and the other person on Discord who had reported this several weeks back is on Windows as well.  Here is the Zendesk report he had filed then, using Muse Choir 0.3.12 - https://musehub.zendesk.com/hc/en-gb/community/posts/17486903847965--Women-and-Full-Choir-Sounds-have-no-sound-above-G4

**Comment by sampleeditor** [comment]
Strange that nothing is playing on "Full Choir" above G4 because the tenors are set to overlap to C5, just like in the "Men"

**Comment by MarcSabatella** [comment]
Still the case for Muse Choir 0.3.15 that just released, both on Windows and Linux (still Muse Sampler 0.6.1.162).

**Comment by DaddyLudwig** [comment]
I seem to be getting just a hint of A4 before it stops, like the A4 is there but can't be sustained for some reason:


https://github.com/musescore/MuseScore/assets/6530761/c12e3910-09bb-4696-bf2e-39814474bbf3

**Comment by MarcSabatella** [comment]
This appears fixed in Muse Choir 0.3.17.  Oddly, this was pushed out to the release channel recently but *not* the beta channel.  However, after removing Muse Choir and reinstalling, I did get 0.3.17 even on beta.

**Comment by sampleeditor** [comment]
@MarcSabatella Did the Women and Full Choir end up working for you with 0.3.17?

**Comment by MarcSabatella** [comment]
@sampleeditor - Yes, as per my comment just above :-).
